tracker/choices.py

```diff
@@ -0,0 +1,23 @@
+from django.db import models
+
+
+class Roles(models.Model):
```

Comment:
We don't need a new model. `Roles` should inherit `models.TextChoices` and inside you should place 
```
PROJECT_LEAD = "L", _("Project-lead")
...
```

Reply:
@Abeeujah Please do also this requested change because your code won't work and will raise `NameError`